profiler/include/profile_grouped_gemm_impl.hpp

```diff
@@ -71,7 +71,7 @@ void profile_grouped_gemm_impl(int do_verification,
             }
         };
 
-    int group_count = Ms.size();
+    std::size_t group_count = Ms.size();
```

Comment:
can do type_convert here.

Reply:
Unchanged as size_t is used in loops, addressed by [57f2d3c](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/57f2d3c31e163a8525856fb3b1cae2cb09ddbff9)